src/main/java/duke/DukeCommands.java

```diff
@@ -0,0 +1,50 @@
+package duke;
+
+/**
+ * The enum duke.Duke commands.
+ */
+public enum DukeCommands {
+    /**
```

Comment:
Perhaps you could remove the individual JavaDocs for every type of commands to reduce the clutter here.